substrate/frame/staking/src/tests.rs

```diff
@@ -38,7 +38,7 @@ use sp_runtime::{
 	Perbill, Percent, Perquintill, Rounding, TokenError,
 };
 use sp_staking::{
-	offence::{DisableStrategy, OffenceDetails, OnOffenceHandler},
+	offence::{OffenceDetails, OnOffenceHandler},
 	SessionIndex,
 };
 use sp_std::prelude::*;
```

Comment:
Can we add unit tests to the implementor of `DisablingStrategy`? 

```rust
mod disabling_strategy {
  #[test]
  // ...

}
```

Reply:
done